packages/manager/modules/pci/src/projects/project/kubernetes/details/node-pool/nodes/add/translations/Messages_fr_FR.json

```diff
@@ -26,5 +26,6 @@
   "kube_nodes_add_select_nodes": "Sélectionnez le nombre de nouveaux nœuds à ajouter",
   "kube_nodes_add_instance_type": "Type d'instance",
   "kube_nodes_add_adding_message": "Redimensionnement du pool",
-  "kube_nodes_add_anti_affinity_active_info": "L'anti-affinité est activée : le pool de nœuds ne pourra pas contenir plus de {{ maxNodes }} noeuds"
+  "kube_nodes_add_anti_affinity_active_info": "L'anti-affinité est activée : le pool de nœuds ne pourra pas contenir plus de {{ maxNodes }} noeuds",
```

Comment:
```suggestion
  "kube_nodes_add_anti_affinity_active_info": "L'anti-affinité est activée : le pool de nœuds ne pourra pas contenir plus de {{ maxNodes }} nœuds",
```

Reply:
Hello @antleblanc, this changes not mine. Please address your request to the original pull request : https://github.com/ovh/manager/pull/3860. I'll make the rebase once done on it